spacy_annotator/__init__.py

```diff
@@ -1,2 +1,257 @@
-from spacy_annotator.pandas_annotations import annotate
-from spacy_annotator.list_annotations import annotate
\ No newline at end of file
+from IPython.display import clear_output, display, display_html
+from ipywidgets import Button, HTML, HBox, Text, Output, Layout
+import pandas as pd
+import spacy
+from spacy import displacy
+from spacy.matcher import PhraseMatcher
+from spacy.tokens import Span
+import warnings
+
+
+class Annotator:
+    """
+    Helper class for SpaCy based NER annotation.
+    Parameters
+    ----------
+    model (spacy model) = SpaCy model for pre-annotation, optional.
+    labels (list): list of named entity labels.
+    delimiter (str): delimiter to separate entities in annotator. Default: ',' (comma).
+    attr (str): include option to skip example while annotating. Default: True.
+    include_skip (bool):
+    """
+
+    def __init__(
+        self,
+        *,
+        model=None,
+        labels,
+        delimiter=",",
+        attr="LOWER",  # "ORTH"
+        include_skip=True,
+    ):
+        self.model = model
+        if self.model is not None:
+            self.nlp = model
+        else:
+            # TODO think of better solution for default?
+            self.nlp = spacy.load("en_core_web_sm")
+        self.labels = labels
+        self.delimiter = delimiter
+        self.attr = attr
+        self.include_skip = include_skip
+
+    @property
+    def instructions(self):
+        print(
+            """
+            \033[1mInstructions\033[0m \n
+            For each entity type, input must be a DELIMITER separated string. \n
+            If no entities in text, leave as is and press submit.
+            Similarly, if no entities for a particular label, leave as is. \n
+            Buttons: \n
+            \t * submit inserts new annotation (or overwrites existing one if one is present). \n
+            \t * skip moves forward and leaves empty string (or existing annotation if one is present). \n
+            \t * finish terminates the annotation session.
+            """
+        )
+
+    def __load_data(self, df, sample_size=1, shuffle=False, strata=None):
+        """
+        Helper function to load data into annotator and pre-process.
+        Parameters
+        ----------
+        df (pandas dataframe): Dataframe with text to be labelled.
+        sample_size (float): Size of the sample to be labelled. Default: 1.
```

Comment:
It would be clearer to call this sample fraction, rather than size (which makes me think of N)